cerbero/build/source.py

```diff
@@ -247,6 +248,11 @@ def extract(self):
 
         # checkout the current version
         git.local_checkout(self.build_dir, self.repo_dir, self.commit)
+        # apply the patches only after checkout to avoid failure with patches already applied.
+        for patch in self.patches:
```

Comment:
Patches is not documented.
patches is not correctly initialized as it's done in Tarball